gdx-pay-android-googlebilling/src/com/badlogic/gdx/pay/android/googlebilling/PurchaseManagerGoogleBilling.java

```diff
@@ -5,10 +5,13 @@
 import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.pay.*;
 
+import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import java.util.*;
 import java.util.concurrent.ConcurrentHashMap;
 
+import static com.badlogic.gdx.pay.android.googlebilling.Iso8601DurationStringToFreeTrialPeriodConverter.convertToFreeTrialPeriod;
```

Comment:
Cosmetics: Avoid static imports

Reply:
Ok I will change.

Reply:
There are some more import static I did not comment on